src/rules/utils.ts

```diff
@@ -592,6 +598,7 @@ const joinNames = (a: string | null, b: string | null): string | null =>
 export function getNodeName(
   node:
     | JestFunctionMemberExpression<JestFunctionName>
+    | TSESTree.TaggedTemplateExpression
     | JestFunctionIdentifier<JestFunctionName>,
```

Comment:
```suggestion
    | JestFunctionIdentifier<JestFunctionName>
    | TSESTree.TaggedTemplateExpression,
```

nit: me appealing to my perfectionist brain that likes the alignment, feel free to ignore 😅

Reply:
i've changed it (: was just trying to reduce noise in the git diff